src/components/mini-widgets/JoystickCommIndicator.vue

```diff
@@ -1,14 +1,18 @@
 <template>
   <div>
-    <div v-tooltip="tooltipText" class="relative cursor-pointer" :class="indicatorClass" @click="showDialog = true">
```

Comment:
Just to better get, what was breaking here? I tried reproducing but saw no bug.

Reply:
When the joystick config page was already open, showing this screen
![image](https://github.com/user-attachments/assets/40e28cfb-7c8a-491a-99f9-cceedf854e29)
And a joystick input was pressed, the whole config screen broke with a vNode error (dom injection incomplete).

The cause was the v-tooltip, when used as a prop, sometimes has bad collateral effects like this one. (Vuetify's fault).


https://github.com/user-attachments/assets/2d4d11ce-c1b7-4e3b-939e-d1b11e0e8292

Reply:
Wow! I actually saw this one already, but had no idea the problem was the indicator! Nice catch!